src/tracer.test.js

```diff
@@ -344,6 +344,20 @@ describe('tracer', () => {
     });
   });
 
+  test('trace; no context', async () => {
+    const token = 'DEADBEEF';
+    const lumigoTracer = require('./index')({ token });
+
+    const userHandler1 = async event => {
+      return 'ok';
+    };
+    const event = { a: 'b', c: 'd' };
+
+    await lumigoTracer.trace(userHandler1)(event);
```

Comment:
Don't you want to check the return value as well?

Reply:
I'm just want to check the wrapping here